packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -22,7 +22,8 @@ function Request(method, urlPath, cache) {
 	this._headers = {};
 	this._timeout = null;
 	this._type = "json"; // superagent's default, only for POST/PUT/PATCH methods
-
+	// CORS off by default
+	this._withCredentials = false;
```

Comment:
To me, the property name `_withCredentials` is vague. One of/variant of the following would be more self-documenting:
- `_enableCORS`
- `_isCORSEnabled`
- `_corsEnabled`
- `_withCORSEnabled`
- `_withCrossOriginHTTPRequestsEnabled`